subsys/net/lib/sntp/sntp_pkt.h

```diff
@@ -9,21 +9,21 @@
 
 #include <zephyr/types.h>
 
-#define LI_MASK   0xC0
-#define VN_MASK   0x38
-#define MODE_MASK 0x07
+#define Z_SNTP_LI_MASK   0xC0
```

Comment:
Is there really an issue for these or are you just thinking some future conflict?
IMHO putting Z_ prefix to these looks superfluous, SNTP_ is enough for these private headers.

Reply:
SNTP_ as a prefix would be better indeed

Reply:
MODE_MASK was immediate conflict, MASK_SHIFT next in queue (literally for tomorrow), then I converted nearby defines for consistency in naming (stopped where changes would overflow to more files).

> IMHO putting Z_ prefix to these looks superfluous, SNTP_ is enough for these private headers.

That's what I'm talking about in https://github.com/zephyrproject-rtos/zephyr/pull/24746#issuecomment-620474250 - we're going to draw any conclusions from the cases we see. All public Zephyr symbols should be clearly namespaced in one way. All private Zephyr symbols should be clearly namespaced in another way (and that way shouldn't be `Z_`, as that's an obvious candidate for public prefix). Until then, we'll have a mess of system, timebomb for naming conflicts happening any time (and all the time).

Reply:
SNTP_ alone won't help. Any general-purpose networking library with the same level of reflection as Zephyr (i.e. poor) would define such symbols, then someone trying to use that library with Zephyr (even non-SNTP parts) would get conflicts.

Reply:
We have quite many symbols in the networking without Z_ but prefixed with some other way. So using SNTP_ prefix here looks just fine. If there is really a conflict at some point, we can start to thing how to solve that.

Reply:
> We have quite many symbols in the networking without Z_ but prefixed with some other way.

So, as I argue all of them, and not just in networking, should be eventually prefixed with a "private prefix", and this PR is the step in the right direction.

> So using SNTP_ prefix here looks just fine.

The question is whether you see Z_SNTP_ as terribly wrong, and why.

> If there is really a conflict at some point, we can start to thing how to solve that.

Yes, but we can solve (or start to solve) them yesterday, not waiting when they start to hit us and users at the least suitable time.

Reply:
> So, as I argue all of them, and not just in networking, should be eventually prefixed with a "private prefix", and this PR is the step in the right direction.

That could be done of course if really needed, I am not seeing the reason why we would start from this library. 

> 
> > So using SNTP_ prefix here looks just fine.
> 
> The question is whether you see Z_SNTP_ as terribly wrong, and why.

IMHO this is just pointless right now. We can change these ones to SNTP_, but putting Z_ is superfluous atm.

Reply:
> That could be done of course if really needed, I am not seeing the reason why we would start from this library.

I argued for proper namespacing ~2 years ago. We're now starting to see sustainable stream of namespacing conflicts (this is 3rd I see in last 2-3 weeks). I put my actions where my words are, by using the currently designated private namespace prefix, when fixing a namespace conflict, instead of just applying a random spur-of-the-moment workaround.

> IMHO this is just pointless right now. We can change these ones to SNTP_, but putting Z_ is superfluous atm.

Ok, to cut this pointless discussion, will remove `Z_`.